tests/vcd_vdc.py

```diff
@@ -1,26 +1,110 @@
-import os
-import unittest
-import yaml
-from pyvcloud.vcd.client import BasicLoginCredentials
-from pyvcloud.vcd.client import Client
+# VMware vCloud Director Python SDK
+# Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyvcloud.vcd.client import TaskStatus
 from pyvcloud.vcd.org import Org
 from pyvcloud.vcd.test import TestCase
 from pyvcloud.vcd.vdc import VDC
+import unittest
 
-class TestVDC(TestCase):
 
+class TestVDC(TestCase):
     def test_list_vdc(self):
-        logged_in_org = self.client.get_org()
-        org = Org(self.client, org_resource=logged_in_org)
+        org_to_use = self.client.get_org()
```

Comment:
I feel passing the resource is the better way here. Since we already have the resource at hand, passing it instead of href prevents getting the resource again in the Org class.

       logged_in_org = self.client.get_org()
        org = Org(self.client, resource=logged_in_org)

Reply:
done

Reply:
when the org data is returned with `self.client.get_org_by_name(name)` the resource is not retrieved, just a summary per org: `<Org href="https://bos1-vcd-sp-static-202-34.eng.vmware.com/api/org/6a2018e8-e119-45ba-a8ac-624d44d335e4" name="org2" type="application/vnd.vmware.vcloud.org+xml"/>`. Therefore we need to get the full org resource data using `get_resource`